packages/mocks/package.json

```diff
@@ -2,11 +2,23 @@
   "name": "@kubev2v/mocks",
   "version": "0.0.1",
   "private": true,
-  "repository": "git@github.com:kubev2v/forklift-console-plugin.git",
   "license": "Apache-2.0",
-  "main": "./src/index.ts",
+  "repository": {
+    "url": "git@github.com:kubev2v/forklift-console-plugin.git",
+    "directory": "packages/mocks"
+  },
+
+  "files": [
+    "./dist/*",
+    "./src/*",
```

Comment:
hmm, that looks bad, can you check why you need it and try to solve the issue without exposing the source ?

Reply:
`files` isn't even needed.  I've made adjustments.